### ISSUE TITLE
Osee Go stream

### ISSUE THREAD

**Issue by trebor-56** [bitfocus/companion-module-requests#1408]
Could be an idea to add to the features/support for the Osee Go stream ?
Thank You

**Comment by alevecchio** [comment]
First we need to wait for a new Osee firmware that support connections on the ethernet port. So far, we are still waiting...

**Comment by henny-0** [comment]
I see that Osee has released the firmware that enables control over a network as well as an unofficial Companion plugin that requires manually adding. I wonder when it will be officially integrated into Companion

**Comment by digitaldrummerj** [comment]
@henny-0 unfortunately for the unofficial Companion module that Osee released it is up to Osee to request a BitFocus repository for the module, put the code there and ask the Companion team to include it in the official release.

**Comment by thedist** [comment]
> henny-0 unfortunately it is up to Osee to request a BitFocus repository for the module, put the code there and ask the Companion team to include it in the official release.

That's not correct. Anyone that wishes to develop a module for Companion can request a repo be created and once working be added to Companion. Most of the modules for Companion are by 3rd parties that have no connection to the manufacturer/developer of a device/app.

If anyone does want to develop a module for it they are welcome to, but would likely need firmware on the device that supports it, and ideally documentation so that they can create a module to interact with the device.

**Comment by digitaldrummerj** [comment]
@thedist totally agree with you.  However, my comment was for the fact that osee has released a module on their site but are only distributing it as a compiled version that they are having us drop it into the bundles modules folder.   To make the Osee release an official Companion bundled module it is up to Osee to do it since only they have the source.

**Comment by thedist** [comment]
If they are distributing a packaged module, that is just PACKAGED, not compiled. Even if they made a custom build of Companion itself including the module again it would still be trivial to get the source and create a repo for it. It does not require Osee it do it themselves.

Would it be nice for the company themselves to do this? sure it'd be great. Is it needed? No, not by any means.

**Comment by henny-0** [comment]
There seems to be something in the works now.

https://github.com/bitfocus/companion-module-osee-gostream

**Comment by ghost** [comment]
Any updates on this?

**Comment by WOTBox** [comment]
It IS being worked on. A fork has been ironing out the issues (mislabelled commands, missing fields etc) in the official add-on module. Soon to be updated on the main https://github.com/bitfocus/companion-module-osee-gostream and then they'll go for inclusion in the next stable Companion public release. No timeframe given. I'd say it's close...

**Comment by leonreucher** [comment]
The module is available in Companion.
https://github.com/bitfocus/companion-module-osee-gostream
